src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -293,33 +295,33 @@ gui.tooltips.appliedenergistics2.MoveWhenWorkIsDone=Auswerfen wenn fertig.
 gui.tooltips.appliedenergistics2.MoveWhenFull=Auswerfen wenn voll.
 gui.tooltips.appliedenergistics2.Disabled=Deaktiviert
 gui.tooltips.appliedenergistics2.Enable=Aktiviert
-gui.tooltips.appliedenergistics2.Blocking=Nicht fertigen, wenn Zielinventar Items enthält.
+gui.tooltips.appliedenergistics2.Blocking=Keine Fertigungsitems einfügen während das Zielinventar Items enthält.
```

Comment:
"Fertigungsitems" is not really a word. At least it should be "Gegenstand" or whatever the official translation for item is used by minecraft.